server/chat-plugins/mafia.ts

```diff
@@ -2566,14 +2572,20 @@ export const commands: ChatCommands = {
 			}
 			this.room = targetRoom;
 			const game = this.requireGame(MafiaTracker);
-			if (game.hostid !== user.id && !game.cohosts.includes(user.id)) this.checkCan('mute', null, targetRoom);
-			const player = game.playerTable[toID(args.join(''))];
-			if (!player) return this.errorReply(`"${args.join(',')}" is not a living player.`);
+			if (game.hostid !== user.id && !game.cohostids.includes(user.id)) this.checkCan('mute', null, targetRoom);
 			if (game.phase === 'IDEApicking') {
 				return this.errorReply(`You cannot add or remove players while IDEA roles are being picked.`); // needs to be here since eliminate doesn't pass the user
 			}
-			game.eliminate(player, cmd);
-			game.logAction(user, `killed ${player.name}`);
+			if (!args[0]) return this.parse('/help mafia kill');
+			for (const targetUsername of args) {
+				const player = game.playerTable[toID(targetUsername)];
+				if (player) {
+					game.eliminate(player, cmd);
+					game.logAction(user, `killed ${player.name}`);
+				} else {
+					user.sendTo(this.room, `|error|${targetUsername} is not a living player.`);
```

Comment:
nit, this can just be `this.errorReply` now (because we set `this.room` to be the proper room)